nsv13/code/modules/munitions/ammunition/missiles/missile_types.dm

```diff
@@ -29,9 +29,19 @@
 		//This one's been claimed!
 		claimable_gulag_points = 0
 
-/obj/item/ship_weapon/ammunition/missile/CtrlClick(mob/user)
+/obj/item/ship_weapon/ammunition/missile/Initialize() ////Sophie: Makes missiles draggable in Zero-G
```

Comment:
Irrelevant to ship PR, please move to separate PR.

Reply:
Separated